engine/lite/embeddedDeploymentManager/src/test/scala/pl/touk/nussknacker/streaming/embedded/StreamingEmbeddedDeploymentManagerTest.scala

```diff
@@ -142,27 +143,27 @@ class StreamingEmbeddedDeploymentManagerTest extends BaseStreamingEmbeddedDeploy
     def message(input: String) = obj("message" -> fromString(input)).noSpaces
     def prefixMessage(prefix: String, message: String) = obj("message" -> fromString(message), "prefix" -> fromString(prefix))
 
-
     fixture.deployScenario(scenarioForOutput("start"))
 
-
-    kafkaClient.sendMessage(inputTopic, message("1")).futureValue
-
-    val consumer = kafkaClient.createConsumer().consumeWithJson(outputTopic)
-    consumer.head shouldBe prefixMessage("start", "1")
+    kafkaClient.sendMessage(inputTopic, message("1"))
```

Comment:
Why did you remove `.futureValue` here and in other places. I see no changes in the signature - it still returns `Future`

Reply:
Ups.. :) Good catch!